contrib/world_builder/source/features/mantle_layer.cc

```diff
@@ -138,7 +138,7 @@ namespace WorldBuilder
                                                                       *(world->parameters.coordinate_system));
 
       if (depth <= max_depth && depth >= min_depth &&
-          Utilities::polygon_contains_point(coordinates, Point<2>(natural_coordinate.get_surface_coordinates(),
+          WorldBuilder::Utilities::polygon_contains_point(coordinates, Point<2>(natural_coordinate.get_surface_coordinates(),
                                                                   world->parameters.coordinate_system->natural_coordinate_system())))
```

Comment:
I think you probably need to indent that.

Reply:
Yep. Figured out I can run the worldbuilder indent. :-)